src/features/identityCheck/pages/identification/__test__/IdentityCheckStart.web.test.tsx

```diff
@@ -17,30 +20,75 @@ jest.mock('features/identityCheck/context/IdentityCheckContextProvider', () => (
   })
 
-  it('should navigate to next screen on press modal continue', () => {
-    const { getByText } = render(<IdentityCheckStart />)
-    expect(mockNavigateToNextScreen).not.toHaveBeenCalled()
+  describe('is not mobile viewport', () => {
+    it(`should navigate to next screen on press "Vérification par smartphone" and "J'ai compris"`, () => {
+      const { getByText } = render(
+        <IdentityCheckStart />,
+        withCustomTheme({ isDesktopViewport: true, isMobileViewport: false })
+      )
+      expect(mockNavigateToNextScreen).not.toHaveBeenCalled()
+
+      fireEvent.click(getByText('Vérification par smartphone'))
+      fireEvent.click(getByText("J'ai compris"))
+      expect(mockNavigateToNextScreen).toHaveBeenCalledTimes(1)
+    })
+
+    it('should navigate to DMS modal when user press "Identification par le site Démarches-Simplifiées"', () => {
+      const { getByText, getByTestId } = render(
+        <IdentityCheckStart />,
+        withCustomTheme({ isDesktopViewport: true, isMobileViewport: false })
+      )
+      expect(mockNavigateToNextScreen).not.toHaveBeenCalled()
 
-    fireEvent.click(getByText('Vérification par smartphone'))
-    fireEvent.click(getByText("J'ai compris"))
-    expect(mockNavigateToNextScreen).toHaveBeenCalledTimes(1)
+      fireEvent.click(getByTestId('Identification par le site Démarches-Simplifiées'))
+      getByText('Je suis de nationalité française')
+      getByText('Je suis de nationalité étrangère')
+    })
   })
 
-  it('should navigate to DMS modal when user choose "Transmettre un document"', () => {
-    const { getByText, getByTestId } = render(<IdentityCheckStart />)
-    expect(mockNavigateToNextScreen).not.toHaveBeenCalled()
+  describe('is mobile viewport', () => {
+    it(`should navigate to next screen on press "Commencer la vérification" and "J'ai compris"`, () => {
+      const { getByText } = render(
+        <IdentityCheckStart />,
+        withCustomTheme({ isDesktopViewport: false, isMobileViewport: true })
+      )
+      expect(mockNavigateToNextScreen).not.toHaveBeenCalled()
 
-    fireEvent.click(getByTestId('Identification par le site Démarches-Simplifiées'))
-    getByText('Je suis de nationalité française')
-    getByText('Je suis de nationalité étrangère')
+      fireEvent.click(getByText('Commencer la vérification'))
+      fireEvent.click(getByText("J'ai compris"))
+      expect(mockNavigateToNextScreen).toHaveBeenCalledTimes(1)
+    })
+
+    it('should navigate to DMS modal when user press "Transmettre un document"', () => {
+      const { getByText, getByTestId } = render(
+        <IdentityCheckStart />,
+        withCustomTheme({ isDesktopViewport: false, isMobileViewport: true })
+      )
+      expect(mockNavigateToNextScreen).not.toHaveBeenCalled()
+
+      fireEvent.click(getByTestId('Transmettre un document'))
+      getByText('Je suis de nationalité française')
```

Comment:
ca sert à quoi ça ?

Reply:
Le `expect(mockNavigateToNextScreen).not.toHaveBeenCalled()` ne sert à rien, mais pour la suite, c'est pour tester que le clique sur "Transmettre un document" redirige bien vers la modale DMS

Reply:
Je ne comprends pas, ici tu ne fais rien. Tu devrais expect